tests/scripts/all.sh

```diff
@@ -2610,14 +2610,11 @@ component_test_no_psa_crypto_full_cmake_asan() {
     msg "test: ssl-opt.sh (full minus PSA crypto)"
     tests/ssl-opt.sh
 
-    msg "test: compat.sh default (full minus PSA crypto)"
-    tests/compat.sh
-
-    msg "test: compat.sh NULL (full minus PSA crypto)"
+    msg "test: compat.sh: NULL (full minus PSA crypto)"
     tests/compat.sh -f 'NULL'
 
-    msg "test: compat.sh ARIA + ChachaPoly (full minus PSA crypto)"
-    env OPENSSL="$OPENSSL_NEXT" tests/compat.sh -e '^$' -f 'ARIA\|CHACHA'
+    msg "test: compat.sh next: all except NULL (full minus PSA crypto)"
```

Comment:
Using `OPENSSL_NEXT` causes static ECDH cipher suites to be skipped. So we need a `compat.sh` invocation with `OPENSSL` for that.

Reply:
Hmm, that's annoying.